src/main/resources/layouts/configuration.fxml

```diff
@@ -18,8 +21,27 @@
 <?import javafx.scene.shape.Line?>
 <?import javafx.scene.text.Font?>
 
-<VBox maxHeight="-Infinity" maxWidth="-Infinity" minHeight="403.0" minWidth="494.0" prefHeight="405.0" prefWidth="546.0" style="-fx-background-color: black;" xmlns="http://javafx.com/javafx/null" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.doubleslash.usb_led_matrix.view.ConfigurationView">
+<VBox maxHeight="-Infinity" maxWidth="-Infinity" minHeight="403.0" minWidth="494.0" prefHeight="405.0" prefWidth="546.0" style="-fx-background-color: black;" xmlns="http://javafx.com/javafx/21" xmlns:fx="http://javafx.com/fxml/1" fx:controller="de.doubleslash.usb_led_matrix.view.ConfigurationView">
   <children>
+    <MenuBar>
+      <menus>
+        <Menu mnemonicParsing="false" text="Busylight">
+          <items>
+            <MenuItem mnemonicParsing="false" onAction="#settingsButton" text="Settings" />
+          </items>
+        </Menu>
+        <Menu mnemonicParsing="false" text="Edit">
+          <items>
+            <MenuItem mnemonicParsing="false" text="Delete" />
+          </items>
+        </Menu>
+        <Menu mnemonicParsing="false" text="Help">
+          <items>
+            <MenuItem mnemonicParsing="false" onAction="#aboutButton" text="About" />
```

Comment:
Please remove the previous button (the large _i_ icon from the UI). Think of:
- resources (image)
- styling (CSS)

Reply:
Done.

Reply:
No, look in the resources folder.

Reply:
Done.